App/ru.lproj/Localizable.strings

```diff
@@ -7,7 +7,7 @@
 */
 
 // MARK: - Dashboard
-"dashboard_tabbar" = "Мониторинг";
+"dashboard_tabbar" = "Панель управления";
```

Comment:
Please check if this string looks good in the beta, because it looks like it might be too long!